src/Morse/MorseCharacter.ts

```diff
@@ -1,14 +1,53 @@
+import EncodingCategory from '../Common/EncodingCategory';
+import EncodingCharacterBase from '../Common/EncodingCharacterBase';
 import MorseData from './MorseData';
-import MorseEntry from './MorseEntry';
-import MorseLookupResult from './MorseLookupResult';
+import MorseEncoding from './MorseEncoding';
+
+class MorseCharacter extends EncodingCharacterBase<MorseEncoding> {
+  public static toMorseString(encoding: MorseEncoding) {
+    let morseChars = '';
+
+    while (encoding !== 0) {
+      if ((encoding & 0b11) === 0b01) {
```

Comment:
Can these magic numbers be stored as constants somewhere?  i.e.
MORSE_DOT = 0b01;
MORSE_DASH = 0b10;
MORSE_BIT_MASK = MORSE_DOT | MORSE_DASH;